packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -111,7 +112,26 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
           </SiteNavNestList>
         </SiteNavItem>
         <SiteNavItem>
-          <SiteNavAnchor to="/tokens">Tokens</SiteNavAnchor>
+          <SiteNavButton onClick={() => setTokensOpen(!tokensOpen)} isOpen={tokensOpen}>
+            Design Tokens
+            <SiteNavAnchorArrow isOpen={tokensOpen} />
+          </SiteNavButton>
+          <SiteNavNestList isOpen={tokensOpen}>
+            <SiteNavItem>
+              <SiteNavAnchor to={SidebarCategoryRoutes.TOKENS}>Token list</SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.TOKENS}/how-to-compose-custom-ui-with-tokens`}>
+                How to compose custom UI with tokens
```

Comment:
<img width="273" alt="image" src="https://user-images.githubusercontent.com/1592327/65535279-b6023680-dec6-11e9-90e3-ea996069bf28.png">
This is kind of long.  Can we rename it to `Composing with tokens`?

Reply:
Seems like an improvement to me but I'll wait for a consensus before changing it as it involves changing filenames and stuff and there maybe other ideas. @aayushpi @serifluous @two24studios for input